app/models/framework/definition/RM3787.fdl

```diff
@@ -36,7 +36,7 @@ Framework RM3787 {
     CustomerName from 'Customer Organisation Name'
     CustomerPostcode from 'Customer Post Code'
     CustomerReferenceNumber from 'Matter Name'
-    ProductDescription from 'Matter Description'
+    String Additional7 from 'Matter Description'
```

Comment:
Should we update the ruby definition to reflect this too?

Reply:
Yes.

Reply:
Well, maybe. If we do it now it takes effect on next deploy. If we don't it takes effect when the FDL supersedes this. I'm guessing you'd prefer the first?

Reply:
Yes, I'd like to have basically no feature changes when we switch from one to that other, and I can add it to the table below and give to Timur.